shell/platform/darwin/ios/ios_context_metal_impeller.h

```diff
@@ -25,8 +25,6 @@ class IOSContextMetalImpeller final : public IOSContext {
 
   ~IOSContextMetalImpeller();
 
-  fml::scoped_nsobject<FlutterDarwinContextMetalSkia> GetDarwinContext() const;
```

Comment:
"Hmm, why is `IOSContextMetalImpeller` returning `FlutterDarwinContextMetalSkia`?" 
It's dead code.